cookbooks/continental_compression_with_imposed_faults/doc/continental_compression_with_imposed_faults.md

```diff
@@ -0,0 +1,56 @@
+
+```{literalinclude} single_fault_imposed.part.wb
+```
+
+# Model Evolution and Potential Expansion.
+
+```{figure-md} fig:strain_rate_and_density_0_myr
+<img src="strain_rate_and_density_0_myr.svg" style="width:50.0%" />
+
+  Strain rate (left) and density distribution (right) at 0 Myr in the upper 100 km from x = 100 to 300 km. Strain localizes in the upper crust along pre-defined fault zones, while density increases with depth, reflecting the compositional and thermal stratification of the lithosphere.
+```
+
+```{figure-md} fig:strain_rate_and_density_35_myr
+<img src="strain_rate_and_density_35_myr.svg" style="width:50.0%" />
+
+ Strain rate (left) and density (right) distributions with temperature contours after 35 Myr of deformation.
+```
+
+Deformation preferentially localizes along the two pre-existing shear zones forming a grabben-like geometry ({numref}`fig:strain_rate_and_density_0_myr`), which reflects their optimal orientation and position within the model domain to accommodate the imposed convergence. After 35 Myr of convergence ({numref}`fig:strain_rate_and_density_35_myr`) deformation remains strongly localized along these faults,  with crustal shortening and thickening occurring between them forming approximately symmetric orogenic wedge. Some degree of asymmetry develops on the right side of the wedge, where the fault farthest to the right remains active in the upper crust and connects to the fault bounding the wedge near the brittle-ductile transition zone.
+
+Given the nonlinearity of the rheology and governing equations, minor variations in fault strength, geometry, lithospheric structure, and boundary velocities may lead to significant variations in the spatiotemporal evolution of deformation. This cookbook provides a flexible framework for exploring the effects of these parameters, and application to hypothesis-driven questions such as fault reactivation, inversion of rift basins, or the partitioning of strain in complex orogens.
+
+These results demonstrate the potential key role of pre-existing faults in guiding the evolution of lithospheric deformation. However, given the nonlinearity of the rheology and governing equations, minor variations in fault strength, geometry, lithospheric structure, and boundary velocities may lead to significant variations in the spatiotemporal evolution of deformation. Similarly, changing the numerical resolution is likely to also affect the results, as the brittle shear band width by introduced by the plastic damper (1e21 Pa s) is still likely not fully resolved at the maximum resolution of 1e21 Pa s. Furthermore, varying degrees of minor variations in the model results will likely occur if a stricter nonlinear solver tolerance is selected. This cookbook provides a flexible framework for exploring the effects of these parameters, and application to hypothesis-driven questions such as fault reactivation, inversion of rift basins, or the partitioning of strain in complex orogens.
```

Comment:
```suggestion
Similarly, changing the numerical resolution is likely to also affect the results, as the brittle shear band width by introduced by the plastic damper (1e21 Pa s) is still likely not fully resolved at the maximum resolution of X km. Furthermore, varying degrees of minor variations in the model results will likely occur if a stricter nonlinear solver tolerance is selected. This cookbook provides a flexible framework for exploring the effects of these parameters, and application to hypothesis-driven questions such as fault reactivation, inversion of rift basins, or the partitioning of strain in complex orogens.
```
I deleted the duplicate text, and I think you meant to put the resolution of the mesh instead of the damper viscosity.